### PR TITLE
OD-692 [Fix] The field type for the Simple List description has been changed to html by default

### PR DESCRIPTION
**Pull request by OleksandrLavrynenko**
@romanyosyfiv

OD-692 https://weboo.atlassian.net/browse/OD-692

## Description
**Problem**: A field type for the description of Simple List is set to be plain text by default but the data contains HTML 
**Solution**: The field type for the Simple List description has been changed to html by default

## Screenshots/screencasts
https://user-images.githubusercontent.com/86256215/163371227-e8e868d6-3dd6-40cf-93f9-6c17e713fe22.mp4

## Backward compatibility
This change is fully backward compatible.

## Reviewers
@upplabs-alex-levchenko

### CODE REVIEW

**Review by tonytlwu — CHANGES_REQUESTED** [review]
You should be able to simple set https://github.com/Fliplet/fliplet-widget-dynamic-lists/blob/6e1ba47fcdfd74218b904654bef52a8123fa0bbe/js/default-configs/layouts-config.js#L348 to `html`

**Comment by OleksandrLavrynenko** [comment]
> You should be able to simple set
> 
> https://github.com/Fliplet/fliplet-widget-dynamic-lists/blob/6e1ba47fcdfd74218b904654bef52a8123fa0bbe/js/default-configs/layouts-config.js#L348
> 
> to `html`
**@tonytlwu** I'm guessing I need to add an object to the "detail-fields"  https://github.com/Fliplet/fliplet-widget-dynamic-lists/blob/6e1ba47fcdfd74218b904654bef52a8123fa0bbe/js/default-configs/layouts-config.js#L362

image
what id should i add?

**Comment by tonytlwu** [comment]
@OleksandrLavrynenko Any random 8 character ID with lower and upper case alphabets should be fine.

File: js/default-configs/layouts-config.js [hunk 1]
```diff
@@ -365,6 +365,13 @@ window.flListLayoutConfig = {
         location: 'Image',
         type: 'image',
         column: 'Image'
+      },
+      {
+        id: 'RGosNfos',
+        interfaceName: 'Secondary text',
```
Comment (tonytlwu) [inline js/default-configs/layouts-config.js:371]: @OleksandrLavrynenko I don't think `interfaceName` is needed. Can you confirm if or why it's necessary?

**Review by tonytlwu — CHANGES_REQUESTED** [review]

File: js/default-configs/layouts-config.js (hunk 1 above)
Comment (OleksandrLavrynenko) [reply js/default-configs/layouts-config.js]: @tonytlwu You're right. This parameter is not needed

**Review by tonytlwu — APPROVED** [review]
@romanyosyfiv @inna-bieshulia This is ready for testing

Risk: Medium
Affected: Simple List LFD layouts
Verify: Detail views are configured with suitable defaults

Squash and merge: Use PR title

**Pull request merged**